Sources/TuistAutomation/Device/DeviceController.swift

```diff
@@ -158,11 +178,24 @@ extension PhysicalDevice {
         case .watchOS: .watchOS
         }
 
+        let transportType: PhysicalDevice.TransportType = switch device.connectionProperties.transportType {
+        case .localNetwork: .wifi
+        case .wired: .usb
+        default: .unknown
+        }
+
+        let connectionState: PhysicalDevice.ConnectionState = switch device.connectionProperties.tunnelState {
+        case .connected: .connected
+        default: .disconnected
+        }
```

Comment:
I'd rewrite these `switch` statements to be exhaustive – that is, I'd remove the `default` case